hathor/verification/verification_service.py

```diff
@@ -80,35 +90,43 @@ def validate_full(
         vertex.set_validation(validation)
         return True
 
-    def verify_basic(self, vertex: BaseTransaction, *, skip_block_weight_verification: bool = False) -> None:
+    def verify_basic(self, base_tx: BaseTransaction, *, skip_block_weight_verification: bool = False) -> None:
```

Comment:
I felt like we are doing a refactor and add a feature in this PR.

Reply:
Which feature?

Reply:
I've removed this from this PR.